helm/idol-nifi/Chart.yaml

```
@@ -33,3 +33,7 @@ dependencies:
   condition: prometheus-adapter.enabled
   version: "4.2.0"
   repository: "https://prometheus-community.github.io/helm-charts"
+- name: postgresql-ha
+  condition: postgresql-ha.enabled
+  version: "14.3.4"
+  repository: https://charts.bitnami.com/bitnami
```

```suggestion
- name: postgresql-ha
  alias: postgresql
  condition: postgresql.enabled
  version: "14.3.4"
  repository: https://charts.bitnami.com/bitnami
```

If you use an alias you should be able to avoid the slightly awkward syntax `(index .Values "postgresql-ha" )`